### PR TITLE
add comments for the functions in the interaction modules

### PR DESCRIPTION
**Pull request by JulienDoerner**
In this PR new comments for the functions of the interaction modules are added.

### CODE REVIEW

File: include/crpropa/module/PhotoPionProduction.h [hunk 1]
```diff
@@ -103,15 +114,42 @@ class PhotoPionProduction: public Module {
...
+
 	void initRate(std::string filename);
+
+	/** get the mean free path for a nucleon
+	 * @param gamma 	Lorentz factor of the nucleon
+	 * @param z 		redshift
+	 * @param onProton 	true for protons, false for neutrons
+	 */
 	double nucleonMFP(double gamma, double z, bool onProton) const;
 	double nucleiModification(int A, int X) const;
```
Comment (lukasmerten) [inline include/crpropa/module/PhotoPionProduction.h:161]: Was the comment for nucleiModification intentionally left out?

File: include/crpropa/module/SynchrotronRadiation.h [hunk 2]
```diff
@@ -53,11 +53,25 @@ class SynchrotronRadiation: public Module {
 	 */
 	SynchrotronRadiation(double Brms = 0, bool havePhotons = false, double thinning = 0, int nSamples = 0, double limit = 0.1);
 	
+	// set the target photon field
 	void setField(ref_ptr<MagneticField> field);
+
+	// set the rms value of the magnetic field (no 3d field is used)
```
Comment (lukasmerten) [inline include/crpropa/module/SynchrotronRadiation.h]: I would write root-mean square (rms) to make it clear.

File: include/crpropa/module/PhotoPionProduction.h (hunk 1 above)
Comment (JulienDoerner) [reply include/crpropa/module/PhotoPionProduction.h:161]: I have added also a comment for the nuclei modification.

File: include/crpropa/module/SynchrotronRadiation.h (hunk 2 above)
Comment (JulienDoerner) [reply include/crpropa/module/SynchrotronRadiation.h]: fixed

**Review by lukasmerten — APPROVED** [review]

**Pull request merged**